src/obr/core/core.py

```diff
@@ -191,3 +196,28 @@ def unlink(fn):
             unlink(fn)
     else:
         unlink(fns)
+
+
+def map_view_folder_to_job_id(view_folder: str) -> dict:
+    """Creates a mapping from from the view schema to the original jobid
+
+    Returns:
+    ========
+        A dictionary with jobid: view_folder
+    """
+    ret = {}
+    base, _, _ = next(os.walk(view_folder))
```

Comment:
Regarding stability, this would throw a `StopIteration` error if the `view_folder` is empty for some reason.